src/python/pants/backend/jvm/tasks/ivy_imports.py

```diff
@@ -28,6 +29,9 @@ def has_imports(target):
         return isinstance(target, ImportJarsMixin) and target.imported_targets
 
     def execute(self):
+        deprecated_conditional(
+            lambda: True, removal_version="1.30.0.dev0", entity_description="The imports goal"
```

Comment:
```suggestion
            lambda: True, removal_version="1.30.0.dev0", entity_description="The `imports` goal"
```

Reply:
Well, they will only see this message if they used `./pants outdated` or `./pants imports`, so they should grok this as-is... If this passes CI I'd rather not trigger another round just for this, but if not I'll make the change along with whatever fixes.

Reply:
Made the changes as well as adding the hints.